### PR TITLE
Wiki updates

### PR DESCRIPTION
**Pull request by AshakaE**
# Brainbot - Wiki updates

_This feature gets wiki activities and posts wikis into channel of choice, the default language at the moment is English_

## How should this be tested?

1. clone the repo
2. checkout `wiki-updates`  branch
3. Add envs
4. npm start

## Notes or observations
_Bot schedule not implemented yet due to having bot post same results each time, probably need to use datetime field as constraint_

## Linked issues

 closes [#243](https://github.com/EveripediaNetwork/issues/issues/243)

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CHANNEL_ID=
+BOT_TOKEN=
+API_URL=
+PAGE_URL
```
Comment (kesar) [inline .env.sample]:
```suggestion
PAGE_URL=
```

File: package.json [hunk 2]
```diff
@@ -1,60 +1,63 @@
...
-    "nodemon": "^2.0.15",
-    "prettier": "^2.6.1",
-    "ts-node": "^10.7.0",
-    "typescript": "^4.6.4"
-  },
-  "engines": {
-    "node": ">=16.0.0",
-    "npm": ">=7.0.0"
-  }
+    "name": "discord.ts-example",
```
Comment (kesar) [inline package.json]:
```suggestion
    "name": "brainbot",
```

File: package.json [hunk 3]
```diff
@@ -1,60 +1,63 @@
...
+        "url": "https://github.com/oceanroleplay/discord.ts-example/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/oceanroleplay/discord.ts-example.git"
+    },
+    "license": "MIT",
+    "contributors": [
+        "Vijay Meena <indianoceanroleplay@gmail.com> (https://github.com/oceanroleplay)"
+    ],
```
Comment (kesar) [inline package.json]: put our brainbot stuff 👍🏻

**Comment by kesar** [comment]
> Bot schedule not implemented yet due to having bot post same results each time, probably need to use datetime field as constraint

yeah, probably do a each min. checking and store as initial timestamp whenever the bot starts, and then each minute you update that timestamp and look for new stuff > old_timestamp 👍🏻

File: readme.md [hunk 4]
```diff
@@ -181,11 +127,3 @@ There are only a few lines of basic code, which you need to either comment out o
 1. Run `npm uninstall koa @koa/router @discordx/koa @types/koa @types/koa__router`
```
Comment (kesar) [inline readme.md]: i didnt full review the code so I might be commenting something wrong, but perhaps we can remove a bunch of dependencies if we are not using rest api server.

also let's make a proper readme file 👍🏻

Comment (AshakaE) [reply readme.md]: okay sure

**Pull request merged**